docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+# Coast to Coast Demo for Q4 2023
+
+This coast to coast demo highlights some of the new features introduced to OLMv1 in Q4 of 2023.
+
+New Features:
+- `Catalog` polling
+- Version range support on the `spec.version` field for `Operator` resources
+- Semver upgrade constraint policy
+- `BundleDeployment` health status
+
+This document will go through an entire coast to coast example that highlights each of these new features. Each step will be documented as if you are following along and will be done sequentially. This document will also use some values specific to this example and in your own projects should be replaced to use a different value.
+
+## Prerequisites
+The following prerequisites are required:
+- A running Kubernetes cluster where you have admin priveleges
```

Comment:
```suggestion
- A running Kubernetes cluster where you have admin privileges.
```

Reply:
With lists like this, the usual guidance is that if you have sentence fragments, you can leave off the period. However, if you use a period once in the list, you should use a period everywhere in the list.

Reply:
Does this include if you use a period anywhere in sentence in the list?

Reply:
[08e8dbb](https://github.com/operator-framework/operator-controller/pull/533/commits/08e8dbba77679d610d04f2b8002613fe94421aff)